cocos/renderer/CCRenderer.cpp

```diff
@@ -640,9 +640,8 @@ void Renderer::render()
     {
         //Process render commands
         //1. Sort render commands based on ID
-        for (auto &renderqueue : _renderGroups)
-        {
-            renderqueue.sort();
+        for (auto renderQueueID : _groupCommandManager->getUsedIDS()) {
+            _renderGroups[renderQueueID].sort();
```

Comment:
Yep, it reduces some sort() invoking for empty vector, but it may bring some performance issue to invoke `_groupCommandManager->getUsedIDS()`.

So i don't know what's the performance improved. And it make things complex.